src/tracer.h

```diff
@@ -72,6 +97,12 @@ class TraceManager {
 
     bool clear_filepath_;
     const std::string* filepath_;
+
+    bool clear_mode_;
+    const InferenceTraceMode* mode_;
+
+    bool clear_config_map_;
+    const TraceConfigMap* config_map_;
```

Comment:
I don't think these new `clear_*` vars are being used. Do they need to be? I believe this is part of the trace extension in http/grpc servers.

If not, I would remove them for now until the feature is added.

Reply:
removed